integrations/k8s-using-deployment/falco.yaml

```diff
@@ -0,0 +1,167 @@
+#
```

Comment:
This file must be ignored, in the instructions you are telling to copy it.

Reply:
No this must be kept for now, it differs from the default one and we have no way of disabling the default rules right now so a specific one seems good.